<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>autorization_mainwindow</class>
 <widget class="QDialog" name="autorization_mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>680</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>668</width>
    <height>428</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>5435235</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>App sale</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>681</width>
     <height>471</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>681</width>
     <height>471</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>681</width>
     <height>471</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
	border: none;
	border-radius: 5px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>40</y>
      <width>671</width>
      <height>56</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #404040</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>206</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding:2px 5px;
height: 28px;
color: white;
background-color: #2a2a2a;
border-radius: 5px;
border: none;
font-size: 14px;</string>
     </property>
     <property name="placeholderText">
      <string>Поиск...</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>355</x>
       <y>10</y>
       <width>111</width>
       <height>36</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	padding: 2px;
	font-size: 14px;
	height: 28px;
	color: white;
	background-color: #2a2a2a;
	border-radius: 5px;
}
QPushButton::hover {
	background-color: #949494;
}</string>
     </property>
     <property name="text">
      <string>Избранное</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>10</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	padding: 2px;
	font-size: 18px;
	height: 28px;
	color: black;
	background-color: #6accc1;
	border-radius: 18px;
}

QPushButton::hover {
	background-color: #55ada3;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>475</x>
       <y>20</y>
       <width>136</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 15px;
color: white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>41</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	padding: 2px;
	font-size: 20px;
	height: 28px;
	color: white;
	background-color: #2a2a2a;
	border-radius: 5px;
}
QPushButton::hover {
	background-color: #949494;
}</string>
     </property>
     <property name="text">
      <string>⋮</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>41</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	padding-top: 2px;
	font-size: 30px;
	height: 28px;
	color: white;
	background-color: #2a2a2a;
	border-radius: 5px;
}
QPushButton::hover {
	background-color: #949494;
}</string>
     </property>
     <property name="text">
      <string>✉</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>265</x>
       <y>10</y>
       <width>41</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	padding-top: 4px;
	font-size: 18px;
	height: 28px;
	color: white;
	background-color: #2a2a2a;
	border-radius: 5px;
}
QPushButton::hover {
	background-color: #949494;
}</string>
     </property>
     <property name="text">
      <string>⭯</string>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>95</y>
      <width>671</width>
      <height>376</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget {
	border: none;
	font-size:15px;
	background-color: #2a2a2a;
	color: white;
	padding: 0px 2px;;
}

QListWidget::item {
	margin-top: 4px;
	padding-left: 5px;
	background-color: #424242;
	height: 35px;
	border: none;
	border-radius: 5px;
}</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>671</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	border-bottom-left-radius: 0px;
	border-bottom-right-radius: 0px;
	background-color: #2a2a2a;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>5</y>
       <width>26</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font-size: 20px;
	height: 28px;
	color: white;
	background-color: #2a2a2a;
	border-radius: 5px;
}
QPushButton::hover {
	background-color: #ffb0be;
	color: black;
}</string>
     </property>
     <property name="text">
      <string>✘</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>5</y>
       <width>26</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	padding-bottom: 4px;
	font-size: 20px;
	color: white;
	background-color: #2a2a2a;
	border-radius: 5px;
}
QPushButton::hover {
	background-color: #949494;
}</string>
     </property>
     <property name="text">
      <string>_</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 14px;
color: white;</string>
     </property>
     <property name="text">
      <string>Sale App</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>625</x>
      <y>420</y>
      <width>36</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	padding: 2px;
	font-size: 20px;
	height: 28px;
	color: black;
	background-color: #6accc1;
	border-radius: 18px;
}

QPushButton::hover {
	background-color: #55ada3;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
